### PR TITLE
SparseML integ for NER

### PR DESCRIPTION
**Pull request by natuan**
SparseML integration for NER task

### CODE REVIEW

File: examples/pytorch/token-classification/sparseml_utils.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from typing import Any
+
+import numpy
+
+from sparseml.pytorch.utils import ModuleExporter
+from transformers.modeling_outputs import TokenClassifierOutput
+from transformers.sparse import SparseMLTrainer
+
+
+class SparseMLTokenClassificationTrainer(SparseMLTrainer):
```
Comment (spacemanidol) [inline examples/pytorch/token-classification/sparseml_utils.py:10]: Isn't this the same behavior as present in sparse.py? Is this needed?

File: examples/pytorch/token-classification/run_ner.py [hunk 2]
```diff
@@ -29,17 +29,18 @@
 from datasets import ClassLabel, load_dataset, load_metric
 
 import transformers
+from sparseml_utils import SparseMLTokenClassificationTrainer, TokenClassificationModuleExporter
```
Comment (spacemanidol) [inline examples/pytorch/token-classification/run_ner.py:32]: Can't we just use the SparseMLTrainer? I see no token classification specific configs

**Review by spacemanidol — CHANGES_REQUESTED** [review]
Is there any reason why not to use the non token classification sparsemltrainer?

File: examples/pytorch/token-classification/sparseml_utils.py (hunk 1 above)
Comment (natuan) [reply examples/pytorch/token-classification/sparseml_utils.py]: Fixed (I planned to refactor these specific trainers across tasks later, but for now just moved the compute_loss here up to SparseMLTrainer, then reused this class)

**Review by spacemanidol — APPROVED** [review]

**Review by bfineran — APPROVED** [review]
LGTM @natuan

**Pull request merged**